lib/ansible/modules/network/vyos/vyos_firewall_global.py

```diff
@@ -0,0 +1,1185 @@
+        type: dict
+        suboptions:
+           address_group:
+             description:
+               - Defines a group of IP addresses for referencing in firewall rules.
+             type: list
+             elements: dict
+             suboptions:
+               name:
+                 description:
+                   - Name of the firewall address group.
+                 type: str
+                 required: True
+               description:
+                 description:
+                   - Allows you to specify a brief description for the address group.
+                 type: str
+               members:
+                 description:
+                   - Address-group members.
+                   - IPv4 address to match.
+                   - IPv4 range to match.
+                 type: list
+                 elements: dict
+                 suboptions:
+                   address:
+                     description: IP address.
+                     type: str
+           network_group:
+             description:
+               - Defines a group of networks for referencing in firewall rules.
+             type: list
+             elements: dict
+             suboptions:
+               name:
+                 description:
+                   - Name of the firewall network group.
+                 type: str
+                 required: True
+               description:
+                 description:
+                   - Allows you to specify a brief description for the network group.
+                 type: str
+               members:
+                 description:
+                   - Adds an IPv4 network to the specified network group.
+                   - The format is ip-address/prefix.
+                 type: list
+                 elements: dict
+                 suboptions:
+                   address:
+                     description: IP address.
+                     type: str
+           port_group:
+             description:
+               - Defines a group of ports for referencing in firewall rules.
+             type: list
+             elements: dict
+             suboptions:
+               name:
+                 description:
+                   - Name of the firewall port group.
+                 type: str
+                 required: True
+               description:
+                 description:
+                   - Allows you to specify a brief description for the port group.
+                 type: str
+               members:
+                 description:
+                   - Port-group member.
+                 type: list
+                 elements: dict
+                 suboptions:
+                   port:
+                     description: IP address.
+                     type: str
+
+      log_martians:
+        description:
+          - Specifies whether or not to record packets with invalid addresses in the log.
+          - (enable) Logs packets with invalid addresses.
```

Comment:
shoudn't this be True/False, otherwise it might confuse the user

Reply:
+1